assets/js/hooks/useMonitorInternetConnection.js

```diff
@@ -54,19 +54,11 @@ export function useMonitorInternetConnection() {
 		}
 
 		try {
-			// Use apiFetch directly, instead of our wrapper API, to have
-			// control over the error handling/catching, as status is switched to offline
-			// if request fails to reach the server.
-			const onlineResponse = await apiFetch( {
-				method: 'GET',
-				path: '/google-site-kit/v1/core/site/data/health-checks',
+			await API.get( 'core', 'site', 'connection-check', undefined, {
+				useCache: false,
 			} );
 
-			// We are only interested if the request was successful, to
-			// confirm online status.
-			const canReachInternetURL = !! onlineResponse.checks;
-
-			setIsOnline( canReachInternetURL );
+			setIsOnline( true );
```

Comment:
We might as well also check the response of this endpoint, since we bother to send one and it's another marker that the connection is good.

It would also help improve the readability of this code I think, because while `API.get` should throw when the request fails, I think seeing a `GET` request without using any of its response makes this code harder to follow, especially when stripped of comments.

So we should either document this a little and/or use the response to verify our connection is good.

Reply:
Good point @tofumatt, I brought back in the use of the response and the code comment too. I tested throwing on the PHP side and the message correctly shows. If the response is valid but not the value true, such as a json response `{ status: 'error' }` the check will still return true. However receiving a response, even if this response is an error, does show the user is online.

Reply:
Hah, good point 👍🏻